cinema/views.py

```diff
@@ -12,6 +20,8 @@
     MovieDetailSerializer,
     MovieSessionDetailSerializer,
     MovieListSerializer,
+    OrderSerializer,
+    OrderListSerializer,
 )
```

Comment:
Code Style: You've grouped the imports correctly, but you have a trailing comma in the last import of each group. In Python, it's a common practice to leave out the trailing comma in the last item of a list or tuple.